exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/vendors/SQLiteDialect.kt

```diff
@@ -156,5 +158,19 @@ open class SQLiteDialect : VendorDialect(dialectName, SQLiteDataTypeProvider, SQ
     companion object {
         /** SQLite dialect name */
         const val dialectName: String = "sqlite"
+        val ENABLE_UPDATE_DELETE_LIMIT by lazy {
+            (DriverManager.getConnection("jdbc:sqlite::memory:")).use { connection ->
+                (connection.createStatement()).use { statement ->
```

Comment:
Consider to replace it with `Transaction.exec()`.

Reply:
It's not suitable to move the val into update/delete functions to catch transaction instance as it's used in test.